cmd/global-heal.go

```diff
@@ -31,7 +31,7 @@ const (
 	leaderLockTimeoutSleepInterval = time.Hour
 )
 
-var leaderLockTimeout = newDynamicTimeout(30*time.Second, time.Minute)
+var leaderLockTimeout = newDynamicTimeout(30*time.Second, 5*time.Second)
```

Comment:
This also increases chattiness, lets keep it a higher value. Like 

```suggestion
var leaderLockTimeout = newDynamicTimeout(1*time.Minute, 30*time.Second)
```